### PR TITLE
Unit Testing Action

### PR DESCRIPTION
**Pull request by cbhernan**
### What does this PR do?
This PR has changes for a super simple unit test. 

Adds a batch file that all contributors should run before pushing to ensure that their updates will pass all github actions.

Adds badges to README to display status of workflows.

### Background info:
Part of our addition of unit test.

### Which issue(s) is/are related to this PR?
https://github.com/CityOfLosAngeles/angeleno-my-account-flutter/issues/13

### CODE REVIEW

**Review by nickchaula — DISMISSED** [review]
GTG approved

File: lib/controllers/api_implementation.dart [hunk 1]
```diff
@@ -38,7 +37,8 @@ class UserApi extends Api {
       throw '$e';
     }
 
-    return user;
+    return User(userId: '', email: '', firstName: '', lastName: '',
```
Comment (nickchaula) [inline lib/controllers/api_implementation.dart:40]: It seems like line 37 will throw an error if it ever fails. In that case, this return will never be called right? No catch/final statement

Comment (cbhernan) [reply lib/controllers/api_implementation.dart:40]: Yes, technically there's no error handling here, but this `fetchUser` method will be replaced with an upcoming branch where we get the User from the Auth0 package so it should be safe to ignore for now - updates to this file were made to pass initial unit test.

**Review by nickchaula — APPROVED** [review]
lgtm approved

**Pull request merged**